pkg/apis/workflow/v1alpha1/workflow_types.go

```diff
@@ -717,10 +717,22 @@ type WorkflowStatus struct {
 	Outputs *Outputs `json:"outputs,omitempty"`
 }
 
+type RetryPolicy string
+
+const (
+	RetryPolicyAlways    RetryPolicy = "Always"
```

Comment:
Simplify the enum
	RetryPolicyAlways    RetryPolicy = "Always" /
	RetryPolicyOnFailure RetryPolicy = "OnFailure"
	RetryPolicyOnError   RetryPolicy = "OnError"